src/tools/pub/cmd/index.md

```diff
@@ -1,132 +1,152 @@
+Pub provides a number of subcommands for managing the
 [packages your code depends on](/tools/pub/dependencies).
 
-In this group, the most commonly used commands are `pub get` and
-`pub upgrade`, which retrieve or upgrade dependencies used by a package.
-Every time you modify a pubspec file, run `pub get`
+In this group, the most commonly used subcommands are `get` and
+`upgrade`, which retrieve or upgrade dependencies used by a package.
+Every time you modify a pubspec file,
+run `dart pub get` or `flutter pub get`
 to make sure the dependencies are up to date. Some IDEs
 perform this step automatically on the creation of a project,
 or any modification of the pubspec.
 
-[`pub cache`](/tools/pub/cmd/pub-cache)
-: Manages pub's local package cache. Use this command to add packages
+[`cache`](/tools/pub/cmd/pub-cache)
+: Manages pub's local package cache. Use this subcommand to add packages
   to your cache, or to perform a clean reinstall of all packages in
   your cache.
 
-[`pub deps`](/tools/pub/cmd/pub-deps)
+[`deps`](/tools/pub/cmd/pub-deps)
 : Lists all dependencies used by the current package.
 
-[`pub downgrade`](/tools/pub/cmd/pub-downgrade)
+[`downgrade`](/tools/pub/cmd/pub-downgrade)
 : Retrieves the lowest versions of all the packages that are
   listed as dependencies used by the current package. Used for testing
   the lower range of your package's dependencies.
 
-[`pub get`](/tools/pub/cmd/pub-get)
+[`get`](/tools/pub/cmd/pub-get)
 : Retrieves the packages that are listed as the dependencies for
   the current package.
   If a `pubspec.lock` file already exists, fetches the version
   of each dependency (if possible) as listed in the lock file.
   Creates or updates the lock file, as needed.
 
-[`pub outdated`](/tools/pub/cmd/pub-outdated)
+[`outdated`](/tools/pub/cmd/pub-outdated)
 : Looks at every package that the current package depends on,
   determines which package dependencies are out of date,
   and gives you advice on how to update them.
-  Use this command when you want to update package dependencies.
+  Use this subcommand when you want to update package dependencies.
 
-[`pub upgrade`](/tools/pub/cmd/pub-upgrade)
+[`upgrade`](/tools/pub/cmd/pub-upgrade)
 : Retrieves the latest version of each package listed
   as dependencies used by the current package. If a `pubspec.lock`
   file exists, ignores the versions listed in the lock file and fetches
   the newest versions that honor the constraints in the pubspec.
   Creates or updates the lock file, as needed.
 
 
-## Running command-line apps
-
-Two commands let you run Dart scripts from the command line:
+### Running command-line apps
 
-* The [`pub run`](/tools/pub/cmd/pub-run) command invokes a Dart script in your
-  package, or in one of its dependencies.
+The [`global`](/tools/pub/cmd/pub-global) subcommand lets you 
+make a package globally available, 
+so you can run scripts from that package’s `bin` directory.
 
```

Comment:
Maybe add `Note this requires adding the pub global directory [to path](https://dart.dev/tools/pub/cmd/pub-global#running-a-script-from-your-path).` ?